libexec/cli/start.exec

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+# 
+# Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
+#
+# This software is licensed under a 3-clause BSD license.  Please
+# consult LICENSE file distributed with the sources of this project regarding
+# your rights to use or distribute this software. 
+#
+# 
+
+
```

Comment:
is there any reason we need to redo code here that is in the [action_argparser](https://github.com/singularityware/singularity/blob/master/libexec/cli/action_argparser.sh)?